src/locale/locales/fr/messages.po

```diff
@@ -6686,7 +6450,7 @@ msgstr "Il y a eu un problème de connexion au serveur"
 #: src/screens/Profile/Header/ProfileHeaderLabeler.tsx:111
 #: src/view/screens/ProfileFeed.tsx:546
 msgid "There was an issue contacting the server, please check your internet connection and try again."
-msgstr ""
+msgstr "Il y a eu un problème de connexion au serveur, vérifiez votre connexion Internet et réessayez."
```

Comment:
Please see previous comment on whether `vérifiez` or `veuillez vérifier` is preferable.